nixos/modules/security/ca.nix

```diff
@@ -111,7 +119,12 @@ in
       # P11-Kit trust source.
       environment.etc."ssl/trust-source".source = "${cacertPackage.p11kit}/etc/ssl/trust-source";
     })
-    { security.pki.caBundle = caBundle; }
+    {
+      security.pki = {
+        inherit cacertPackage;
+        inherit caBundle;
```

Comment:
(nit) you can also do `inherit cacertPackage caBundle;`.
Normally, I wouldn't point out such things, but there's another remark, so a second iteration is necessary anyways.